packages/components/src/text/Text.styles.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { text } from './mixins';
+
+export const Text = styled.p(
+	`margin: 0;`,
```

Comment:
In an "application" the `margin` is generally a pain and is frequently overridden in many cases. Conforming to BEM-like principles, letting the parent "block" provide the spacing e.g. a `<Card>` component will probably have a `<Card.Title>` component where the margin is dictated.